src/test/suite/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+import Mocha from "mocha";
+import glob from "glob";
+
+export function run(): Promise<void> {
```

Comment:
I don't see any need for this in https://mochajs.org/

Reply:
I believe it's a good compromise because it uses the same "structure" as the [suggested way](https://code.visualstudio.com/api/working-with-extensions/testing-extension) to test vscode extensions.
In the future will be easier to reuse if we want to ever use vscode testing api.
If you don't like this idea, I can change it to use the "not-implementing" mode.

Reply:
I think we can add it when needed. We don't have any plans currently to test the entire extension and it should be easy to add it at that time.

Reply:
Simplified in the latest commit and merged with the latest main version!